service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java

```diff
@@ -101,22 +101,24 @@
     private SendDocumentService sendDocumentService;
     @Mock
     private FeatureToggleService featureToggleService;
+    @Mock
+    private CaseData caseData;
+
     @InjectMocks
     private AdditionalApplicationsUploadedEventHandler underTest;
 
-    private static final CaseData CASE_DATA = mock(CaseData.class);
-    private static final DocumentReference TEST_DOCUMENT = mock(DocumentReference.class);
     private static final Long CASE_ID = 12345L;
-
     private static final String EMAIL_REP_1 = "email-rep1@test.com";
     private static final String EMAIL_REP_2 = "email-rep2@test.com";
     private static final Set<String> EMAIL_REPS = new HashSet<>(Arrays.asList(EMAIL_REP_1, EMAIL_REP_2));
     private static final String DIGITAL_REP_1 = "digital-rep1@test.com";
     private static final String DIGITAL_REP_2 = "digital-rep2@test.com";
+    private static final String SUBJECT_LINE = "Lastname, SACCCCCCCC5676576567";
     private static final Set<String> DIGITAL_REPS = new HashSet<>(Arrays.asList(DIGITAL_REP_1, DIGITAL_REP_2));
     private static final List<Element<Other>> NO_RECIPIENTS = Collections.emptyList();
     private static final List<Element<Other>> SELECTED_OTHERS = List.of(element(mock(Other.class)));
```

Comment:
there is a static mock here, do we want to remove this one as well?

Reply:
this is rather harmless, as there is not behaviour recorder for it nor verification.